omod/src/main/webapp/resources/htmlforms/liberia/section-ncd.xml

```diff
@@ -0,0 +1,1040 @@
+<!--
+  ~ The contents of this file are subject to the OpenMRS Public License
```

Comment:
I'll defer to @lnball  on this one.

Reply:
Are you asking about the License comment @mogoodrich ?  We don't have any consistent process on this.  Are there guidelines for OpenMRS?

Reply:
No @lnball, sorry... the diff suggested this was a whole new file.

Reply:
@BranfordTGbieor This is a new "section" for Liberia?  I didn't think you were using sections for Liberia, but we do use these for Haiti NCD.  Don't you use https://github.com/PIH/openmrs-config-pihliberia/blob/master/configuration/pih/htmlforms/ncd-adult-initial.xml and followup?

Reply:
I made no changes to this file. Guess there's some problem with my local repo; been tryinna delete it for  some time now. There should only be the files, scripts and styles directories in the resource dir and not an htmlform dir. Will try to delete again.

Reply:
Cool @BranfordTGbieor  looks like you fixed it, the file is now gone!